pkg/ingester/ingester.go

```diff
@@ -708,7 +714,18 @@ func New(cfg Config, limits *validation.Overrides, registerer prometheus.Registe
 		logger:                       logger,
 		ingestionRate:                util_math.NewEWMARate(0.2, instanceIngestionRateTickInterval),
 		expandedPostingsCacheFactory: cortex_tsdb.NewExpandedPostingsCacheFactory(cfg.BlocksStorageConfig.TSDB.PostingsCache),
+		matchersCache:                storecache.NewNoopMatcherCache(),
+	}
+
+	if cfg.MatchersCacheMaxItems > 0 {
+		r := prometheus.NewRegistry()
```

Comment:
Why do we need a new registry? We can just use `registerer`

Reply:
Because i'm renaming the metrics.

I dont wanna this metrics to have the `thanos` name. WDYT?